### DISCUSSION TITLE
Ability to disable AudioContext option

### DISCUSSION THREAD

**Discussion by down-bad** [ciderapp/Cider#1207] [Feature Request]
A recent commit force-enables the AudioContext option by default, with no ability to disable it, which creates audio problems on my system. There is absolutely no reason why that should be.

I would love to see this matter resolved so I can continue to use Cider.

**Comment by maikirakiwi** [comment]
Fixed in commit https://github.com/ciderapp/Cider/commit/e28b484d9224fca060e8642dbdd2f44ede99956f
